Sources/BraintreeCoreSwift/BTKeychain.swift

```diff
@@ -0,0 +1,76 @@
+import Foundation
+import Security
+
+@objcMembers public class BTKeychain: NSObject {
```

Comment:
Same [documentation question](https://github.com/braintree/braintree_ios/pull/825/files#r914133820) for this file. May be irrelevant if merchants don't touch these.